diffraction/WISH/bragg-detect/cnn/README.md

```diff
@@ -0,0 +1,21 @@
+Bragg Peaks detection using a Faster RCNN model
+================
+
+Inorder to use the pretrained Faster RCNN model inside mantid below steps are required.
+
+* Install mantid from conda `mamba create -n mantid_cnn -c mantid mantidworkbench`
+* Activate the conda environment with `mamba activate mantid_cnn` 
+* Launch workbench from `workbench` command
+* Download the script repository's `scriptrepository\diffraction\WISH` directory as instructed here https://docs.mantidproject.org/nightly/workbench/scriptrepository.html 
+* Close the workbench
+* From command line, change the directory to the place where the scripts were downloaded ex: LocalScriptRepo\diffraction\WISH 
+* Within the same conda enviroment, install pytorch dependancies by running `pip install -r requirements.txt`
+* Install NVIDIA CUDA Deep Neural Network library (cuDNN) by running `conda install -c anaconda cudnn`
+* Re-launch workbench from `workbench` command
```

Comment:
Missing step here to add path to bragg-detect in the manageUserDIrectories > Script directpries tab

Reply:
If import is not working - add troubleshooting guide